### PR TITLE
[CH] Fix GlutenLiteralExpressionSuite and GlutenMathExpressionsSuite

### PR DESCRIPTION
**Pull request by taiyang-li**
## What changes were proposed in this pull request?

(Please fill in changes proposed in this fix)

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)

(If this patch involves UI changes, please attach a screenshot; otherwise, remove this)

### CODE REVIEW

**Comment by taiyang-li** [comment]
cc @baibaichen

File: gluten-ut/spark33/src/test/scala/org/apache/spark/sql/catalyst/expressions/GlutenMathExpressionsSuite.scala [hunk 1]
```diff
@@ -18,11 +18,40 @@ package org.apache.spark.sql.catalyst.expressions
...
 import org.apache.spark.sql.GlutenTestsTrait
 import org.apache.spark.sql.catalyst.dsl.expressions._
 import org.apache.spark.sql.types._
 
+import org.apache.commons.math3.util.Precision
+
+import java.nio.charset.StandardCharsets
+
 class GlutenMathExpressionsSuite extends MathExpressionsSuite with GlutenTestsTrait {
+  override protected def checkResult(
```
Comment (PHILO-HE) [inline gluten-ut/spark33/src/test/scala/org/apache/spark/sql/catalyst/expressions/GlutenMathExpressionsSuite.scala:31]: Hi @taiyang-li, is this check also applied to Velox backend testing?

Comment (taiyang-li) [reply gluten-ut/spark33/src/test/scala/org/apache/spark/sql/catalyst/expressions/GlutenMathExpressionsSuite.scala:31]: Yes, I'll apply it only to CH backend.

**Review by PHILO-HE — APPROVED** [review]
Thanks!

**Pull request merged**